v2/cmd/asoctl/go.mod

```diff
@@ -7,102 +7,102 @@ replace github.com/Azure/azure-service-operator/v2 => ../../
 replace github.com/Azure/azure-service-operator/v2/tools/generator => ../generator
 
 require (
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.6.0
-	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.3.0
-	github.com/Azure/azure-service-operator/v2 v2.0.0-00010101000000-000000000000
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.8.0-beta.2
```

Comment:
Should we really be using `-beta.2`? I would think we want the latest stable version